variables.tf

```diff
@@ -117,3 +117,13 @@ variable "monitoring_role_arn" {
   description = "The ARN for the IAM role that permits RDS to send Enhanced Monitoring metrics to CloudWatch Logs"
   default     = ""
 }
+
+variable "storage_encrypted" {
+  description = "Specifies whether the DB instance is encrypted."
+  default     = true
```

Comment:
Could you please change this to string "true" rather than using a boolean.  The reason for this is that future / older version may convert types differently but a string can always be compared with a string safely.